static-assets/components/cstudio-admin/mods/graphiql.js

```diff
@@ -45,21 +43,28 @@ YAHOO.extend(CStudioAdminConsole.Tool.GraphiQL, CStudioAdminConsole.Tool, {
 			actions = [];
 
 		CStudioAuthoring.Service.lookupConfigurtion(
-			CStudioAuthoringContext.site, 
-			"/environment/environment-config.xml", {
+			CStudioAuthoringContext.site,
+			'/environment/environment-config.xml', {
 				success: function(config) {
-					graphServerUrl = config['graphql-server-url'];
-					GraphiQL(document.getElementById("graph-container"), graphServerUrl + '/api/1/site/graphql', site);
+          graphServerUrl = config['graphql-server-url'];
```

Comment:
Looks like this is creating a global variable. Don't see a `const`/`let` statement where graphServerUrl is defined.